### PR TITLE
Readme updates

### PR DESCRIPTION
**Pull request by pfraze**
Updates the screenshot, gives a better sales pitch, and gives clearer install instructions.

### CODE REVIEW

**Comment by pfraze** [comment]
@dominictarr @mixmix RFR

**Comment by pfraze** [comment]
& @clehner

File: README.md [hunk 1]
```diff
@@ -2,41 +2,70 @@
...
 ![screenshot](./screenshot.png)
 
-Patchwork is a decentralized sharing app.
-It was built for friends of the SSB project, to test basic functions.
-We use it as a test-bed for features, and for our own daily messaging.
+**Patchwork is a decentralized messaging and sharing app.**
 
-Because SSB is a research project, **you'll need to contact an SSB team member (in #scuttlebutt on Freenode) to get onto the network!**
-That's our informal barrier to entry right now, since we're not prepared for lots of users yet.
+It's better than email because:
```
Comment (mixmix) [inline README.md]: I think in a future iteration I'd like to see why email is the gold standard currently. 
I think the average person doesn't see that 

@pfraze

File: README.md [hunk 2]
```diff
@@ -2,41 +2,70 @@
...
+
+ - Private messages are end-to-end encrypted, always.
```
Comment (mixmix) [inline README.md]: / by default, and requires no complex setup

File: README.md [hunk 3]
```diff
@@ -2,41 +2,70 @@
...
+ - You have to follow somebody to get messages from them, so you won't get spammed.
```
Comment (mixmix) [inline README.md]: The network communicates along lines of trust, so if someone breaks trust they can no longer abuse other users

**Comment by mixmix** [comment]
nice one ! :pineapple:

File: README.md (hunk 1 above)
Comment (pfraze) [inline README.md]: You think? Almost everybody uses email.

**Comment by pfraze** [comment]
Gonna go ahead and merge

**Pull request merged**